python/lsst/ap/association/dia_object.py

```diff
@@ -203,10 +187,24 @@ def get_light_curve(self):
         """
 
         # Loop through DIASources and return the "light curve"
-        # I'm keeping this separate from coupte summary statistics for the
-        # moment.
+        # Right now I'm making this the same as returning the
+        # dia_source_catalog.
+
+        return self.dia_source_catalog()
```

Comment:
This should raise `NotImplementedError` rather than returning something that you later plan on changing.